assets/src/scripts/Components/Combobox.jsx

```diff
@@ -0,0 +1,255 @@
+import { SelectorIcon, XIcon } from "@heroicons/react/solid";
```

Comment:
Thinking about naming..

This is a codelist combobox, not a generic combobox. If I've understood correctly, we couldn't reuse it in a form to pick something else. Is it worth renaming it to make it more explicit now, so as not to confuse someone who wants to use the component in future, or wait until we have another use case?

Reply:
Yeah I went back and forth on this. Ideally we would pass all the information via props, and it would be a generic component, but that would make the component much more complex now, when it's only used in one location.

I think it would probably be best moving this to a more generic component when we have a second use case – but happy to change it now if that is less confusing.

Reply:
I think the component is fine as it is, until we get a second (or third) use case. It was just the generic name that I thought might trip people up. 

Happy for you to leave it as is though